crates/lib3h/src/transport/protocol.rs

```diff
@@ -68,14 +68,18 @@ pub type DynTransportActor = Box<
         TransportError,
     >,
 >;
+
+//type UserData = TransportEncoding;
```

Comment:
I don't think this comment should be here...

Reply:
Removed.